### PR TITLE
Reorganizes events  page, adds missing events

### PR DESCRIPTION
**Pull request by Faxmashine**
Also removes `OnSpawn` because it's deprecated

### CODE REVIEW

File: Tools/Docusaurus/docs/Events.md [hunk 1]
```diff
@@ -5,76 +5,146 @@
...
-| OnDisable | OnPreCull | Update |
-| OnEnable | OnPreRender | |
+- FixedUpdate
+- LateUpdate
+- OnCollisionEnter2D
+- OnAnimatorIK
+- OnAnimatorMove
+- OnAudioFilterRead
+- OnBecameVisible
+- O- nBecameInvisible
```
Comment (ScruffyRules) [inline Tools/Docusaurus/docs/Events.md:17]: Random key inputs here?

Comment (Faxmashine) [reply Tools/Docusaurus/docs/Events.md:17]:
```suggestion
- OnBecameInvisible
```

**Comment by momo-the-monster** [comment]
@Faxmashine - I like how you've broken up the Udon events. With the Unity events in a single long list, they take up a lot more space now, and are probably less useful for regular reference - how about we move them to appear after the Udon events?

**Comment by Faxmashine** [comment]
I've moved it to the end of the document.

It was initially a table, which was more difficult to maintain. But it was more compact!
I don't believe Docusaurus has a way for us to automate or replicate that.

**Review by momo-the-monster — APPROVED** [review]

**Pull request merged**